loadgen/loadgen.cc

```diff
@@ -697,7 +697,8 @@ struct PerformanceSummary {
     QuerySampleLatency value = 0;
   };
   /// \todo Make .90 a TestSetting and update relevant hard-coded strings.
-  PercentileEntry latency_target{.90};
+  PercentileEntry latency_target_multistream{settings.latency_percentile_multistream};
+  PercentileEntry latency_target_server{settings.latency_percentile_server};
```

Comment:
For the PerformanceSummary, you can just keep one latency_target and use a conditional here if needed.